extruder/extrude_recipes.py

```diff
@@ -419,8 +419,13 @@ def main(args=None):
         parser.add_argument('--template-dir', default=TEMPLATE_FOLDER,
                             help="Path the folder of recipe templates, if "
                                  "any. Default: '{}'".format(TEMPLATE_FOLDER))
+        parser.add_argument('--dont_copy_conda_forge', action='store_true',
```

Comment:
I think the usual approach is to use dashes in the option name, so `--dont-copy-conda-forge`. Up to you whether to change it though, since hopefully it won't be needed much :)